README.md

```diff
@@ -94,6 +94,14 @@ Have something to contribute or discuss? [Open a pull request](https://github.co
         </ul>
       </td>
     </tr>
+    <tr>
+      <td>bats (Bash automated testing system)</td>
+      <td>
+        <ul>
+          <li><a href="https://github.com/filmil/bazel-bats">filmil/bazel-bats</a></li>
```

Comment:
Could you add this into the Shell row instead?

Reply:
Will do when able.

Reply:
Done.